docs/k0sctl-install.md

```diff
@@ -0,0 +1,171 @@
+INFO [ssh] 10.0.0.1:22: generating token       
+INFO [ssh] 10.0.0.2:22: writing join token     
+INFO [ssh] 10.0.0.2:22: installing k0s worker  
+INFO [ssh] 10.0.0.2:22: starting service       
+INFO [ssh] 10.0.0.2:22: waiting for node to become ready 
+INFO ==> Running phase: Disconnect from hosts 
+INFO ==> Finished in 2m2s                
+INFO k0s cluster version 0.10.0 is now installed  
+INFO Tip: To access the cluster you can now fetch the admin kubeconfig using: 
+INFO      k0sctl kubeconfig              
+```
+
+And, -- presto your k0s cluster is up and running
+
+Get kubeconfig:
+```
+$ k0sctl kubeconfig > kubeconfig
+$ KUBECONFIG=kubeconfig kubectl get pods -A
+NAMESPACE     NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-5f6546844f-w8x27   1/1     Running   0          3m50s
+kube-system   calico-node-vd7lx                          1/1     Running   0          3m44s
+kube-system   coredns-5c98d7d4d8-tmrwv                   1/1     Running   0          4m10s
+kube-system   konnectivity-agent-d9xv2                   1/1     Running   0          3m31s
+kube-system   kube-proxy-xp9r9                           1/1     Running   0          4m4s
+kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
+```
+
+### Upgrade k0s version on your setup
+
+_NOTE: Upgrade will only work if you previously used k0sctl v0.4.0 and you are using v0.5.0_
+
+The previous example installed k0s v0.10.0 on one controller and one worker.
+Let's upgrade the cluster to `v0.11.0-beta.2` 
+Note: to get most recent version please visit release [page](https://github.com/k0sproject/k0s/tags).
+
+In the k0sctl.yaml change this yaml tag:
+```
+  k0s:
+    version: 0.11.0
+```
+
+```
+$ k0sctl apply --config k0sctl.yaml 
+
+⠀⣿⣿⡇⠀⠀⢀⣴⣾⣿⠟⠁⢸⣿⣿⣿⣿⣿⣿⣿⡿⠛⠁⠀⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀█████████ █████████ ███
+⠀⣿⣿⡇⣠⣶⣿⡿⠋⠀⠀⠀⢸⣿⡇⠀⠀⠀⣠⠀⠀⢀⣠⡆⢸⣿⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀███          ███    ███
+⠀⣿⣿⣿⣿⣟⠋⠀⠀⠀⠀⠀⢸⣿⡇⠀⢰⣾⣿⠀⠀⣿⣿⡇⢸⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠀███          ███    ███
+⠀⣿⣿⡏⠻⣿⣷⣤⡀⠀⠀⠀⠸⠛⠁⠀⠸⠋⠁⠀⠀⣿⣿⡇⠈⠉⠉⠉⠉⠉⠉⠉⠉⢹⣿⣿⠀███          ███    ███
+⠀⣿⣿⡇⠀⠀⠙⢿⣿⣦⣀⠀⠀⠀⣠⣶⣶⣶⣶⣶⣶⣿⣿⡇⢰⣶⣶⣶⣶⣶⣶⣶⣶⣾⣿⣿⠀█████████    ███    ██████████
+
+INFO[0000] k0sctl 0.5.0 Copyright 2021, Mirantis Inc.   
+INFO[0000] Anonymized telemetry will be sent to Mirantis. 
+INFO[0000] By continuing to use k0sctl you agree to these terms: 
+INFO[0000] https://k0sproject.io/licenses/eula          
+INFO[0000] ==> Running phase: Connect to hosts                   
+INFO[0000] [ssh] 10.0.0.17:22: connected              
+INFO[0000] [ssh] 10.0.0.23:22: connected            
+INFO[0000] ==> Running phase: Detect host operating systems 
+INFO[0000] [ssh] 10.0.0.23:22: is running CentOS Linux 7 (Core) 
+INFO[0000] [ssh] 10.0.0.17:22: is running CentOS Linux 7 (Core) 
+INFO[0000] ==> Running phase: Prepare hosts   
+INFO[0000] ==> Running phase: Gather host facts 
+INFO[0000] [ssh] 10.0.0.23:22: discovered bond0 as private interface      valid_lft forever preferred_lft forever 
```

Comment:
```suggestion
INFO[0000] [ssh] 10.0.0.23:22: discovered bond0 as private interface      valid_lft forever preferred_lft forever 
```

Hmm, I wonder what happened here 🤔

Reply:
I'm not sure what you mean by this? I grabbed the output as it is

Reply:
Yes and it's alarming. It comes from:
```go
					log.Infof("%s: discovered %s as private interface", h, iface)
```

So where the heck did the `valid_lft forever preferred_lft forever` come from 🤔

Reply:
I will leave it as it is now. If you think it should be removed i will have it removed

Reply:
![image](https://user-images.githubusercontent.com/224971/109623359-bdab1d80-7b45-11eb-9759-b2e936aed1c0.png)

It does not happen in the smoke tests.